### PR TITLE
Entity xml

### PR DESCRIPTION
**Pull request by gislenius**

### CODE REVIEW

**Comment by gislenius** [comment]
This was a real mess. It's now cleaned up. I kept some fields for backwards incompatibility.

File: queries/public/api/entities.jq [hunk 1]
```diff
@@ -63,7 +62,8 @@ let $entities :=
   for $entity in $entities
   return {|
     project($entity, "_id")[$profile-name eq "sec"],
-    { EID: $entity._id }[empty($entity.EIDs)],
+    { ID: $entity._id }[$profile-name eq "sec"],
```
Comment (gislenius) [inline queries/public/api/entities.jq]: That's just to be backwards compatible.

File: queries/public/api/entities.jq [hunk 2]
```diff
@@ -63,7 +62,8 @@ let $entities :=
...
+    { EIDs: [ $entity._id ] }[empty($entity.EIDs)],
```
Comment (gislenius) [inline queries/public/api/entities.jq]: That's to make EIDs consistent with all profiles.

File: queries/public/api/entities.jq [hunk 3]
```diff
@@ -83,6 +83,21 @@ let $entities :=
     trim($entity, "_id")
   |}
 
+let $entities :=
```
Comment (gislenius) [inline queries/public/api/entities.jq]: That's just backwards compatible stuff as well.

File: queries/public/api/filings.jq [hunk 4]
```diff
@@ -90,21 +89,28 @@ let $summaries :=
...
-          NumAbstracts: archives:num-abstracts($archive),
-          NumConcepts: archives:num-concepts($archive)
-      }
+          Profiles: {
+            SEC: {|
+              { Name: "SEC" },
+              filings:summaries($archive)
+            |}
+          },
+          AccessionNumber: $archive._id
```
Comment (gislenius) [inline queries/public/api/filings.jq]: That's just to be backwards compatible.

File: queries/public/api/filings.jq [hunk 5]
```diff
@@ -119,7 +125,7 @@ let $serializers := {
         switch($profile-name)
         case "sec" return
           <Filings>{
```
Comment (gislenius) [inline queries/public/api/filings.jq]: Will be renamed to Archives at some point, but it will need a heads-up to Charlie because it would break.

**Comment by wcandillon** [comment]
@gislenius very cool :+1:

**Pull request merged**